tests/test_rebase_consensus.py

```
@@ -0,0 +1,897 @@
+class TestRebaseWithConsensusIntegration:
+    """Tests for consensus paths inside rebase_with_conflict_resolution()."""
+
+    @pytest.mark.asyncio
+    @patch("sova.git.rebase._create_providers")
+    @patch("sova.git.rebase._load_consensus_config")
+    @patch("sova.git.rebase._configure_diff3", new_callable=AsyncMock)
+    @patch("sova.git.rebase.run", new_callable=AsyncMock)
+    async def test_consensus_disabled_with_fewer_than_two_models(
+        self,
+        mock_run: AsyncMock,
+        mock_diff3: AsyncMock,
```

Ruff (0.16.1)

[warning] Unused method argument: `mock_diff3`

(ARG002)